generate_clouds_yaml.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -xe
+
+source common.sh
+source utils.sh
+source network.sh
+source ocp_install_env.sh
+
+generate_auth_template
```

Comment:
I think this script could be really useful, maybe it's worth adding some notes on the documentation about it?

Reply:
Good point, I added a docs update